.github/workflows/mac-builds.yml

```diff
@@ -42,11 +42,11 @@ jobs:
 
     - name: Build tests + lib
       working-directory: ${{runner.workspace}}/build
-      run: make -j 2
+      run: make -j 3
```

Comment:
```suggestion
      run: make -j `sysctl -n hw.ncpu`
```

Reply:
Cool! In my projects I just use `-j`, which executes **all** in parallel, but those are pretty small (less than 20 source files), so the risk of exhausting available memory is negligible.